ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -294,6 +301,15 @@ int getOperationsCount() {
   }
 
   /**
+   * Return an approximate count of the number of background operations submitted to the router that are not yet
```

Comment:
The reason is borrowed from `currentOperationsCount` which may not reflect an accurate number as certain operations fail before they actually get submitted and yet we increment this number (because the decrement happens in `completeOperation`). But I guess it is more confusing than helpful, so will remove the term - and we can say that incrementing this variable is when we consider an operation as active.